.gitignore

```
@@ -261,3 +261,5 @@ result*
 
 # node_modules
 node_modules/
+
+connector_auth.toml
```

Do you think prefixing a `**` would help? It would prevent a file with name `connector_auth.toml` in any directory (including the root) from being committed.

```suggestion
**/connector_auth.toml
```